frontends/election-manager/src/components/zero_report_print_button.tsx

```diff
@@ -54,27 +53,30 @@ export function ZeroReportPrintButton(): JSX.Element {
     });
   }
 
+  const zeroReport = (
+    <ElectionManagerTallyReport
+      election={election}
+      fullElectionExternalTallies={[]}
+      fullElectionTally={emptyFullElectionTally}
+      isOfficialResults={false}
+    />
+  );
+
   return (
     <React.Fragment>
       <PrintButton
         afterPrint={logZeroReportPrintSuccess}
         afterPrintError={logZeroReportPrintError}
         sides="one-sided"
+        printTarget={zeroReport}
+        printTargetTestId="zero-report"
       >
```

Comment:
Really cool to see how you've combined `PrintButton` and `PrintableArea`, making the print interface even more intuitive!